dspace-server-webapp/src/main/java/org/dspace/app/rest/converter/RootConverter.java

```diff
@@ -25,7 +25,7 @@ public RootRest convert() {
         RootRest rootRest = new RootRest();
         rootRest.setDspaceName(configurationService.getProperty("dspace.name"));
         rootRest.setDspaceURL(configurationService.getProperty("dspace.url"));
-        rootRest.setDspaceRest(configurationService.getProperty("dspace.restUrl"));
```

Comment:
not sure if it is a good idea to setup this properties in the test. In this way we are not really checking that our default settings are what we expect. For instance, this test would fail to note if the properties are missing in the default dspace.cfg file.
Essentially here we are testing only that the rest endpoint return the value of the expected properties but removing this initialization and relying on our default value we will also check that are default are in place

Reply:
after further thinking I realised that this is not an Integration Test but an unit test. So I have added a check for our use case in the `org.dspace.app.rest.RootRestResourceControllerIT`